plugins/dataTypes/Boolean/schema.json

```diff
@@ -0,0 +1,11 @@
+{
+	"title": “Boolean”,
```

Comment:
I just used an existing schema.json file and changed the names in it. So I guess in the original json file the same error is in. I will check it.